robot_ws/src/object_tracker_robot/scripts/run_turtlebot_controller.sh

```diff
@@ -2,5 +2,9 @@
 
 set -ex
 
+DOWNLOAD_MODEL_PYTHON_PATH="$(dirname "$0")/download_model.py"
+CONFIG_YAML_PATH="$(dirname "$0")/../../share/object_tracker_robot/config/model_config.yaml"
 MODEL_PATH="$(dirname "$0")/../../share/object_tracker_robot/model.pb"
+
 python3 -m robomaker.inference_worker ${MODEL_PATH}
+python ${DOWNLOAD_MODEL_PYTHON_PATH} ${CONFIG_YAML_PATH} ${MODEL_PATH}
```

Comment:
I believe downloading the model needs to happen before launching the inference worker.

Reply:
Yes, @mm318, You are right. We have removed run_turtlebot_controller.sh as per @AAlon suggested and merged inference_worker and download_model as one node. So, Model download will happen before starting inference_worker.

Please find [PR#41](https://github.com/aws-robotics/aws-robomaker-sample-application-objecttracker/pull/41) for more details.